cmd/pillard/midware/auth.go

```diff
@@ -0,0 +1,39 @@
+package midware
+
+import (
+	"github.com/anvilresearch/go-anvil"
+
+	"github.com/ardanlabs/kit/cfg"
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+)
+
+const cfgAnvilHost = "ANVIL_HOST"
+
+// Auth handles token authentication.
+func Auth(h app.Handler) app.Handler {
```

Comment:
Should we move this to a new location that can be shared between all apis in this repo?  I propose we put them into /internals/middleware as they are not packages meant to be shared outside this paradigm but will be useful for all shelf apis.